scheds/rust/scx_flash/src/bpf/main.bpf.c

```diff
@@ -1228,6 +1329,40 @@ static void preempt_curr(s32 cpu)
 	bpf_rcu_read_unlock();
 }
 
+/*
+ * Enqueue a task when running in round-robin mode.
+ */
+static void rr_enqueue(struct task_struct *p, struct task_ctx *tctx,
+		       s32 prev_cpu, u64 enq_flags)
+{
+	/*
+	 * Try to bounce the task on another CPU if it has been re-enqueued
+	 * due to a higher scheduling class stealing the CPU.
+	 */
+	if (enq_flags & SCX_ENQ_REENQ) {
+		scx_bpf_dsq_insert(p, SCX_DSQ_GLOBAL, task_slice(prev_cpu), enq_flags);
```

Comment:
It might be nice to try to keep it sticky to the LLC if possible

Reply:
Absolutely, I've used  `SCX_DSQ_GLOBAL` for simplicity, but this part of the round-robin mode can be definitely improved. I'll  add this "locality" improvement in another PR. Thanks @hodgesds!